packages/components-react/projects/jsdom-polyfill/src/lib/a11y-4587e563.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
These files within jsdom-polyfill shouldn't be committed.

Reply:
Removed these files, they were committed occasionally during merge